docs/specs/forecast-api.md

```diff
@@ -176,16 +160,13 @@ Coordinates:
   * latitude   (latitude) float32 3kB -80.0 -79.75 -79.5 ... 79.5 79.75 80.0
   * flight_level      (flight_level) int16 2B 300
   * time       (time) datetime64[ns] 8B 2024-07-01T12:00:00
+    forecast_reference_time (time) datetime64[ns] 8B 2024-07-01T06:00:00
```

Comment:
A tiny question for `time` and `forecast_reference_time`: Is it intentional to remove the timezone `Z` from the otherwise ISO8601 timestamp?

Reply:
great question. I believe time-like data is actually stored as int64 in the netCDF -- the types here reflect the marshaling that xarray does when importing the file, so the timezone naive type here is auto-magically set by xarray.

@mlshapiro can you check me on correctness here?

Reply:
That does bring up a good questions as to whether we'd want to suffix `Z` on the timestamps returned in the geoJSON properties, however: https://github.com/contrailcirrus/contrail-forecast/blob/refactor/v1-signature/docs/specs/forecast-api.md#response-object-1

As it is now, we have consistency. Timezone naive timestamps throughput; implicit UTC.

Adding suffixes in the Propeties of the geoJSON may create mixed contentions, but add specificity to the geoJSON response, at least.

I could see arguments for either way. But I leaning towards consistency (and of course, there is always documentation).

Reply:
Also noting that the `timestamp` passed as a query param (`timestamp="%Y%m%d%H"`) is timezone naive, UTC implicit as well.

If we wanted to be timezone aware here... we'd probably want to go to `timestamp=<int>` unixtime or `timestamp=<iso_datestring>`.  I see that being more canonical in general... but we loose the self-documenting goodness of the current design (which communicates to customers that response data only exists on the hour).

Reply:
> We might change the forecast to exist more often than only once per hour in the future.

Reply:
1. Snapping to the nearest forecast would be perfect. This would allow us to be flexible in what timestamps we accept which IMO makes the API easily usable. Callers can then learn which actual forecast was provided in the response through the reference timestamp dimension.

2. No strong opinion, your proposal sounds good.